apps/src/music/progress/MusicValidator.ts

```diff
@@ -17,34 +17,102 @@ export interface ConditionNames {
 }
 
 export const MusicConditions: ConditionNames = {
-  PLAYED_SOUNDS_TOGETHER: {name: 'played_sounds_together', valueType: 'number'},
+  PLAYED_SOUNDS_TOGETHER: {
+    name: 'played_sounds_together',
+    valueType: 'number',
+    description:
+      'Successful when a minimum number of sounds were playing simultaneously. Ex. Value: 2',
```

Comment:
Given the way we evaluate and present validation, I'd change all of these to present tense. Maybe: 
> Successful when at least this many sounds are playing simultaneously.